integreat_compass/cms/constants/roles.py

```diff
@@ -0,0 +1,28 @@
+"""
+This module contains the possible names of roles to make them translatable and the permission definitions.
+"""
+from django.utils.translation import gettext_lazy as _
+
+BOARD_MEMBER = "BOARD_MEMBER"
+INTEGRATION_SPECIALIST = "INTEGRATION_SPECIALIST"
+OFFER_PROVIDER = "OFFER_PROVIDER"
+
+#: Choices for non-staff roles
+CHOICES = [
+    (BOARD_MEMBER, _("Board Member")),
+    (INTEGRATION_SPECIALIST, _("Integration Specialist")),
+    (OFFER_PROVIDER, _("Offer Provider")),
+]
+
+BOARD_MEMBER_PERMISSIONS = ["view_offer_votes", "vote_on_offer"]
+
+INTEGRATION_SPECIALIST_PERMISSIONS = ["favorite_offer", "comment_on_offer"]
+
+OFFER_PROVIDER_PERMISSIONS = ["change_offer", "upload_mediafile"]
```

Comment:
The integreat-cms has proven that managing permissions in migrations is quite cumbersome, I wouldn't recommend it for a new system.
A different approach would be to manage them via fixtures, see https://github.com/digitalfabrik/integreat-cms/issues/1857